shed/event_streams.py

```diff
@@ -1135,7 +1164,9 @@ def __init__(self, lossless, *children):
                                 self.special_docs_names}
         self.lossless = lossless
         self.lossless_buffer = deque()
-        EventStream.__init__(self, children=children)
+        # Keep track of the emitted docuement types
+        self.lossless_emitted = set()
```

Comment:
This fixed an annoying bug which caused multiple emission as different documents came down.